packages/logger/tests/unit/Logger.test.ts

```diff
@@ -8,14 +8,17 @@ import { context as dummyContext } from '../../../../tests/resources/contexts/he
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
 import * as dummyEvent from '../../../../tests/resources/events/custom/hello-world.json';
-import { LambdaInterface } from '../../examples/utils/lambda';
 import { createLogger, Logger } from '../../src';
 import { EnvironmentVariablesService } from '../../src/config';
 import { PowertoolLogFormatter } from '../../src/formatter';
 import { ClassThatLogs } from '../../src/types';
-import { Context } from 'aws-lambda';
+import { Context, Handler } from 'aws-lambda';
 import { Console } from 'console';
 
+interface LambdaInterface {
+  handler: Handler
+}
+
```

Comment:
It's needed for the decorator unit test cases, Tracer [has one like this as well](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/packages/tracer/tests/unit/Tracer.test.ts#L13-L15). 

Before this PR it used to be in a separate file inside the now deleted folder. I have added it to this file instead of living a folder with one single file of 3 lines.